kube-burner-test/config-podman.yaml

```diff
@@ -0,0 +1,11 @@
+log:
+  level: debug
+logged_outputs:
+  error:
+    level: error
+deployer:
+  type: docker
```

Comment:
should this be podman?

Reply:
Good catch. I must have missed hitting save.

Reply:
Fixed.